script.js

```diff
@@ -1,4 +1,447 @@
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  console.log("L1GameMode", L1GameMode);
+  if (input == "refresh") {
+    myOutputValue = refresh();
+  } else if (L1GameMode == L1_PICK_GAME_MODE) {
+    myOutputValue = startGame(input);
+  } else if (L1GameMode == L1_NORMAL_MODE) {
+    myOutputValue = normal(input);
+  } else if (L1GameMode == L1_LOWEST_MODE) {
+    myOutputValue = lowestWin(input);
+  } else if (L1GameMode == L1_AUTO_MODE) {
+    myOutputValue = auto(input);
+  } else if (L1GameMode == L1_VARIABLE_DICE_MODE) {
+    myOutputValue = variableDice(input);
+  } else if (L1GameMode == L1_VARIABLE_PLAYERS_MODE) {
+    myOutputValue = variablePlayers(input);
+  }
+  // console log to track which L1 mode we are current in
+  console.log("L1GameMode after", L1GameMode);
+  console.log("arrayForNums", arrayForNums);
+  return myOutputValue;
+};
+
+//____________________________ Mode to select which game to play________________
+
+// Start game function where after user choose their game mode (Normal, Lowest Win, Variable Dice), we will set the L1 mode. L2 mode will prime the code to know which step to take.
+
+var startGame = function (input) {
+  var myOutputValue = `Please type in "Normal", "Lowest Win", "Auto", "Variable Dice", "Variable Player", "Knockout" `;
+  if (input == "Normal") {
+    L1GameMode = L1_NORMAL_MODE;
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>2 Players will take turns to roll the dice 🎲.<br><br>After the roll, each player may choose the order to form their number.<br><br>Player with the highest number wins.<br><br>Click Submit to start rolling!`;
+    L2GameMode = L2_ROLL_DICE;
+  }
+  if (input == "Lowest Win") {
+    L1GameMode = L1_LOWEST_MODE;
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>2 Players will take turns to roll the dice 🎲.<br><br>After the roll, each player may choose the order to form their number.<br><br>Player with the lowest number wins.<br><br>Click Submit to start rolling!`;
+    L2GameMode = L2_ROLL_DICE;
+  }
+  if (input == "Auto") {
+    L1GameMode = L1_AUTO_MODE;
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br>2 Players will take turns to roll the dice 🎲.<br><br>After the roll, Computer will choose the best order for player.<br><br>Player with the highest number wins.<br><br>Click Submit to start rolling!`;
+    L2GameMode = L2_ROLL_DICE_AUTO;
+  }
+  if (input == "Variable Dice") {
+    L1GameMode = L1_VARIABLE_DICE_MODE;
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br> Start by choosing the number of 🎲.`;
+    L2GameMode = L2_CHOOSE_NUM_DICE;
+  }
+  if (input == "Variable Players") {
+    L1GameMode = L1_VARIABLE_PLAYERS_MODE;
+    myOutputValue = `You have selected ${input} as your game of choice! <br><br> Start by choosing the number of players ⛹🏻‍♀️⛹🏻‍♂️.`;
+    L2GameMode = L2_CHOOSE_NUM_DICE;
+  }
+  return myOutputValue;
+};
```

Comment:
Excellent. Good job affecting the global variables (game modes) via this function. In the context of game, perhaps having 1 word max as input would prevent typos from the user